calblueprint/ogsc
# Add SAT, ACT, HS graduation, college admissions fields

* Adds fields for SAT, ACT, HS graduation info, college admission info
* Fun fact: we exceeded some limit that TS holds on expanding type unions, so a couple new type errors were corrected to account for the larger ProfileFieldKey enum (or unions computed from this enum)

## How to Test/Use PR feature

* Visit the create player form and validate the new entry options for SAT, ACT, HS graduation info, and college admission info
* Validate that the new test scores appear for admins and shared profiles, validate that HS grad/college admission info appears in all profiles

## PR Checklist

Does your branch (check if true):

## Migrations

* 20210430062311-AddTestingAndGraduationFields: Adds new enum members to ProfileFieldKey

CC: @sydbui
